### ISSUE TITLE
API - .Net Standard Support

### ISSUE THREAD

**Issue by rsbavaresco** [Microsoft/RTVS#4308]
Hello,

I would like to reference `Microsoft.R.Host.Client.API` from .Net Core project, it means, .Net Standard Library. Is it possible?

In time, is there any plan to publish `Microsoft.R.Host.Client.API` in .Net Standard 2.0?

I also asked at https://github.com/MikhailArkhipov/RTVS-cs/issues/3 with more infos.

Thank you all

**Comment by MikhailArkhipov** [comment]
Yes, there is a plan. There is no strict date, might be around May-June.

**Comment by MikhailArkhipov** [comment]
See https://github.com/Microsoft/RTVS/issues/3726
